app/(main)/graduation-progress/[studentCode]/[department]/page.tsx

```
@@ -220,19 +230,21 @@
-            requiredCredits={area.required_credits}
-            isCompleted={area.earned_credits >= area.required_credits}
-            requiredElectiveCredits={area.required_elective_courses}
-            courses={area.courses}
-          />
-          {index < areaProgress.length - 1 && <div className="gap-12"></div>}
-        </div>
-      ))}
+      {areaProgress
+        .filter(area => area.required_credits > 0)
+        .map((area, index) => (
+          <div key={area.area_type}>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)